config/__init__.py

```diff
@@ -26,4 +26,4 @@ def read_database_url_config() -> str:
 
 DATABASE_URL = read_database_url_config()
 
-STATIC_FOLDER = 'arlo-client/public/' if os.environ.get('FLASK_ENV') == 'test' else 'arlo-client/build/'
+STATIC_FOLDER = os.path.normpath(os.path.join(__file__, '..', '..', 'arlo-client', 'public' if os.environ.get('FLASK_ENV') == 'test' else 'build'))
```

Comment:
Was this fixing a specific issue or just preventative? Just curious

Also this line is getting a bit confusing to read for me - do we have any formatting standards that might help either break it up onto multiple lines with indentation or break it up into multiple statements

Reply:
This was actually required, and I don't have a very clear sense why. However, I think it accidentally worked before because the docs say it should be an absolute path. So now it is.

Formatting is one of the issues I'd like to tackle at some point. There are a few different competing options for python formatting, and I don't have a good sense which to use.

Reply:
I don't know the formatting options too well. The last codebase I was working on used one and had an autoformatter and wow did that save some mental energy! A whole aspect of coding I never had to think about again. So I'm excited about that now. I'd be happy to pick one randomly or just let someone else who knows more decide